htdocs/fichinter/class/fichinterligne.class.php

```diff
@@ -337,17 +337,49 @@ public function update_total()
 
 			dol_syslog("FichinterLigne::update_total", LOG_DEBUG);
 			$resql = $this->db->query($sql);
-			if ($resql) {
-				$this->db->commit();
-				return 1;
-			} else {
+			if (!$resql) {
 				$this->error = $this->db->error();
+				$error++;
+			}
+
+			if (!$error && isModEnabled('ticket')) {
+				// Get linked tickets
+				require_once DOL_DOCUMENT_ROOT . '/fichinter/class/fichinter.class.php';
+				$intervention = new Fichinter($this->db);
+				$intervention->id = $this->fk_fichinter;
+				$intervention->fetchObjectLinked(null, "ticket", null, '', 'OR', 1, 'sourcetype', 0);
+				if (!empty($intervention->linkedObjectsIds["ticket"])) {
+					// Update tickets duration
+					$sql = "UPDATE llx_ticket AS t1";
```

Comment:
Can you confirm this king of request is compatible with pgsql ?
May be a loop in PHP would make code easier to understand and more compatible ?